src/mercator/mercator/static/js/Packages/Mercator/2016/Workbench/Workbench.ts

```diff
@@ -197,7 +197,15 @@ export var proposalListingColumnDirective = (
                         ]
                     }];
 
-                    if (currentPhase === "result") {
+                    if (currentPhase === "evaluate") {
```

Comment:
In evaluate, only the "shortlist" filter is availabl. In result, "winner" and "community award" arw added and "winner" is selected by default.